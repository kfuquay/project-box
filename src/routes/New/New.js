import React from "react";
import ProjectForm from '../../components/ProjectForm/ProjectForm'

function New() {
  return(
    <div className="main-container">
      <h3 className="large-text">New Project</h3>
      <ProjectForm params={{ edit: 'N' }} />
    </div>
  )
}

export default New;
