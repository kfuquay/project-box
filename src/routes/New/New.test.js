import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import New from './New'

describe(`New component`, () => {
  it("renders without crashing", () => {
    const wrapper = shallow(<New />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
