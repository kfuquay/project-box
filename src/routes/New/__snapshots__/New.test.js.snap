// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`New component renders without crashing 1`] = `
<div
  className="main-container"
>
  <h3
    className="large-text"
  >
    New Project
  </h3>
  <ProjectForm
    params={
      Object {
        "edit": "N",
      }
    }
  />
</div>
`;
